cfgov/ask_cfpb/tests/models/test_pages.py

```diff
@@ -719,13 +720,14 @@ def test_status_string(self):
         test_redirect_page = self.create_answer_page(answer_base=answer2)
         test_page.redirect_to = test_redirect_page.answer_base
         self.assertEqual(
-            test_page.status_string.lower(), "redirected but not live")
+            test_page.status_string.lower(),
+            _("redirected but not live").lower())
         test_page.live = True
         self.assertEqual(
-            test_page.status_string.lower(), "redirected")
+            test_page.status_string.lower(), _("redirected").lower())
         test_page.redirect_to = None
         self.assertEqual(
-            test_page.status_string.lower(), "live")
+            test_page.status_string.lower(), _("live").lower())
```

Comment:
@higs4281 want to make you aware of this change. All of these strings we're testing are wrapped in `_()` in `status_string()`. Wagtail actually includes a spanish translation of `live`. Because there are Spanish pages that `activate('es')` because their `language` is set to `"es"` earlier in the test run, Spanish is ending up being the active language when this test is run. So, this test was failing with `"live" != "ver en vivo"`. This change should ensure that if we ever provide translations for the other strings this test will not fail, no matter what language might be selected for running the tests.

Reply:
This test doesn't care about language, and adding the ugettext prefix should be fine.